docs_src/index.html.md

```diff
@@ -68,6 +68,11 @@ please feel free to
 </aside>
 
 # Getting Started
+> Node.js users can install the CLI using [npm](https://www.npmjs.com/get-npm): 
+
+```shell
+npm install -g inertia-cli
+```
```

Comment:
Can we put this after the `brew` instructions?

Reply:
I put it first cuz npm is not platform specific and I think it's more common than the rest, but sure thing I can reorder it 👍

Reply:
Homebrew is [crossplatform as well now](https://brew.sh/2019/02/02/homebrew-2.0.0/) - though now that I thikn about it, if anyone tries to use the [current formula](https://github.com/ubclaunchpad/homebrew-tap/blob/master/inertia.rb) on another platform it'll break horrible 😋 

> Homebrew officially supports Linux and Windows 10 with Windows Subsystem for Linux (WSL).

Reply:
That's super new, right? Just a month ago it was MacOS only 😛